src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx

```diff
@@ -27,25 +27,19 @@ import {
   usePermissions,
 } from '@manuscripts/style-guide'
 import { TrackChangesStatus } from '@manuscripts/track-changes-plugin'
-import { ApolloError } from 'apollo-client'
 import debounce from 'lodash.debounce'
 import React, { useEffect, useMemo } from 'react'
 import styled from 'styled-components'
 
 import config from '../../../config'
 import { useCreateEditor } from '../../../hooks/use-create-editor'
 import useTrackedModelManagement from '../../../hooks/use-tracked-model-management'
-import {
-  graphQLErrorMessage,
-  Person,
-  useGetPermittedActions,
```

Comment:
I don't think there is a good reason not to delete these graphql things. We are planning to stop direct contact from graphql server anyway.

Reply:
I did a clean-up there, for unused queries